openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -98,7 +98,7 @@ def __init__(self):
 
             # we have to make sure the toolkit can be loaded before formatting this dict
             # Note any new file write formats should be added here only
-            self._toolkit_file_write_formats = {
+            self._toolkit_file_write_formats: dict[str, Any] = {  # type: ignore[assignment]
```

Comment:
Can we get rid of this type ignore comment by changing the annotation in the base class from `list[str]` to `iterable[str]`?

Reply:
These changes intersect with #1833 which might have made this out of date - I tried to make them more consistent there